quickstarts/prometheus-agent/config.yml

```diff
@@ -0,0 +1,38 @@
+slug: prometheus-agent
+
+description: |
+  ## About New Relic Prometheus Agent
+  
+  New Relic created the [Prometheus Configurator](https://github.com/newrelic/newrelic-prometheus-configurator) in order to simplify the experience of configuring features like discovery, filtering, metrics decoration, and sharding. 
+  
+  This configurator generates a configuration file that is used to run a Prometheus Server in Agent mode to later send the metrics to the New Relic Remote Write Endpoint.
```

Comment:
I would replace "This configurator" by "It"

Reply:
Updated here dc417d4add7042ba507b7f98f7d123c67cd2f44b